packages/tools/client-logger/app-insights-logger/src/test/app/App.tsx

```diff
@@ -0,0 +1,276 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
```

Comment:
it's a bit weird to have a app in test folder. Should it be separate sub-module or package?

Reply:
It's a sample to showcase how one would integrate the logger into their own app, so several people in our team felt that it living with the thing it showcases makes sense. I do admit that (to me) it feels pretty big for a sample app (separate topic), but given its purpose I'd still keep it here. A separate package or repo feels a bit overkill; not worth splitting this and adding to our package sprawl.